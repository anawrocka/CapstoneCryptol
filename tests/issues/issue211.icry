take`{1} (join [0x1, 0x2, error "boom", 0x4])
take`{1} [True, False, error "boom", True]
(splitAt`{0} [True, False, error "boom", True]).0
(splitAt`{0} [True, False, error "boom", True]).1
