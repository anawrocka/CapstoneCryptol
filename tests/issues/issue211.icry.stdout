Loading module Cryptol
0x0
0x1
0x0

Run-time error: boom
